style.css

```diff
@@ -0,0 +1,143 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;500;700&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+  text-decoration: none;
+  list-style: none;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+body {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+
+  /* Identical to box height, or 11% */
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  /* Primary /B400 */
+  color: #6070ff;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+nav ul {
+  float: right;
+}
+
+nav ul li {
+  display: inline-block;
+}
+
+nav ul li a {
+  padding: 0 14px;
+  margin: 0;
+  color: #fff;
+}
+
+.checkbtn {
+  color: #6070ff;
+  cursor: pointer;
+  width: 40px;
+  height: 40px;
+  float: right;
+  border-radius: 8px;
+}
+
+#check {
+  display: none;
+}
+
+.container {
+  background-image: url("./images/header-shapes\ mobile.png");
+  background-position: absolute;
+  height: 587.15px;
+  left: -34px;
+  top: 104px;
+}
```

Comment:
- [ ] Kindly make sure headline introduction contents are aligned correctly in order to match the provided Figma design as much as possible. I highly recommend aligning this section using `flexbox` which gives you a lot of options to align all our elements, plus, it is very easy to manage the app responsiveness.

- [ ] Kindly make sure to add the `border-radius` on the left-bottom of your background image in order to match the provided Figma design.

    - ### YOUR DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/234327672-d9f9d0d1-a1c0-493e-8740-6a6fd343da2c.png)

    - ### FIGMA DESIGN
    
    ![image](https://user-images.githubusercontent.com/87197412/234328258-a43381b2-5bed-4bb3-b153-a883b4c4b923.png)